app/components/search-result.js

```diff
@@ -27,6 +27,16 @@ export default Ember.Component.extend(Analytics, {
         return result.description.slice();
     }),
 
+    shortContributorList: Ember.computed('result', function() {
+        let filtered = this.set('justContributors', this.get('result.contributors').filter(item => !!item.users.bibliographic));
```

Comment:
In general, we try to avoid creating a computed property that has side effects (changes other things)- this can lead to odd behavior that's hard to debug.

Instead, `justContributors` should be a separate computed that `shortContributorsList` watches instead of `results`.

Perhaps it would be more efficient to stop by and pair program to make sure we nail this down? I'm free most of the afternoon.